crates/linker/src/pe.rs

```diff
@@ -574,17 +581,25 @@ struct Preprocessor {
 impl Preprocessor {
     const SECTION_HEADER_WIDTH: usize = std::mem::size_of::<ImageSectionHeader>();
 
-    fn preprocess(output_path: &Path, data: &[u8], extra_sections: &[[u8; 8]]) -> MmapMut {
-        let this = Self::new(data, extra_sections);
-        let mut result = open_mmap_mut(output_path, data.len() + this.additional_length);
+    fn preprocess(
+        output_path: &Path,
+        data: &[u8],
+        dummy_dll_symbols: usize,
+        extra_sections: &[[u8; 8]],
+    ) -> MmapMut {
+        let this = Self::new(data, dummy_dll_symbols, extra_sections);
```

Comment:
I guess rust doesn't let you name a variable self, thus it is this?

Reply:
yes, exactly. `this` is used by rust-analyzer when it inlines, so I'm guessing that is the standard alternative